internal/support/notifications/sending_service.go

```diff
@@ -141,3 +143,70 @@ func handleFailedTransmission(t models.Transmission) {
 		}
 	}
 }
+
+// validateLine checks to see if a line has CR or LF as per RFC 5321
+func validateLine(line string) error {
+	if strings.ContainsAny(line, "\n\r") {
+		return errors.New("smtp: A line must not contain CR or LF")
+	}
+	return nil
+}
+
+func sendMail(addr string, a mail.Auth, from string, to []string, msg []byte, skipVerify bool) error {
```

Comment:
Working on it.

Reply:
Agreed, but first I have re-named this function to smtpSend, and the original smtpSend to sendMail, basically swapped the names, because the smtpSend sounded like the name of a low level function, while sendMail sounded like a bit higher level function, I was constantly getting confused. I could equally keep them as is, but I liked them this way better.

I have moved SmtpInfo related processing to now called (smtpSend) by passing the SmtpInfo as a parameter, that led to shortening of parameter list. The processing of message content is still in the sender function (sendMail) because I thought it would be good to keep protocol communication separate from the message that is being sent. smtpSend handles the protocol part, while sendMail handles message processing and other things related to support-notification feature. Auth is now completely handled within the (now) smtpSend function.

Reply:
I forgot to update the comment, it had only a nominal comment, expanded it to a more detailed one and updated the branch.